client/src/Components/AdminEditProject.js

```diff
@@ -4,10 +4,64 @@ import { signout } from './AuthenticationHelper'
 
 class AdminCreateProject extends Component{
 
+    state={
+        project_name:'',
+        project_max_size:'',
+    }
+
+    componentDidMount(){
+        fetch('http://localhost:6060/api/admin/projects', {
+            credentials: 'include',
+            headers: { 'Content-Type': 'application/json' },
+        })
+        .then(response => response.json())
+        .then(j => {
+            for(var i=0; i<j.length;i++)
+            {
+                if(j[i].id==this.props.match.params.project_id)
+                {
+                  console.log(this.props.match.params.project_id);
+                  this.setState({ 
+                      project_name:j[i].name,
+                      project_max_size:j[i].max_group_size,
+                  });
+                }
+            }
+            
+        })
+    }
+
+
+    setMaxGroupSize(e) {
+        if(e.target.value>=0)
+        this.setState({ project_max_size: e.target.value });
+    }
+
     HandleCencelBtn()
     {
         this.props.history.goBack();
     }
+
+    //NEED TO FIX
+    HandleEditBtn()
+    {
+        const jsonString = JSON.stringify({ name: this.state.project_name, max_group_size: this.state.max_group_size });
+        fetch('http://localhost:6060/api/admin/projects/' + this.props.match.params.project_id, {
+            method:'PUT',
+            credentials: 'include',
+            headers: { 'Content-Type': 'application/json',
+            'Accept': 'application/json', 
+            'Access-Control-Allow-Origin': "*",
+            'Access-Control-Allow-Methods': "PUT"},
```

Comment:
This backend route does not exist, you currently **cannot** update a project via the front end

Reply:
I am ok to delete these, is it mean that the method: PUT and method: DELETE can not be used in the project.